app/system/libs/combodate/js/combodate.js

```diff
@@ -228,6 +228,10 @@
             var items = [], name, i,
                 longNames = this.options.template.indexOf('YYYY') !== -1;
 
+            var year = new Date().getFullYear();
+            var yearName = longNames ? year : ( year + '').substring(2);
+            items[this.options.yearDescending ? 'push' : 'unshift']([year, yearName]);
```

Comment:
Pretty sure this will put the current year at the bottom in one of the two cases of yearDescending